onnxruntime/test/common/rocm_op_test_utils.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
```

Comment:
maybe don't add this file - 
leverage USE_ROCM or something like:
bool enable_rocm = (nullptr != DefaultRocmExecutionProvider().get()) 

if in future checking version is required, we can do:
#if defined(USE_CUDA) || (defined(USE_ROCM)  && ROCM_VERSION >= 40300)